deploy/deploy.sh

```diff
@@ -1,26 +0,0 @@
-#!/bin/bash
```

Comment:
Removing this because there is no functional difference between this and hack/deploy.sh ... at the moment I really only want the CI to go green ... so cleaning up hack/ and deploy/ can probably wait.

Reply:
I'm more in favor of removing `hack/deploy.sh` because `deploy/deploy.sh` has use when people want to `curl` for a non-olm deployment and `hack/deploy.sh` is really for in repo testing of non-olm deployments, which is uncommon.

Reply:
The problem is that hack/deploy.sh is what is used in CI that's running on these PRs.

Reply:
right openshift 3.11....  Let me see if the CI team can support 4.1/4.2 so we can drop this script.